x-pack/plugins/synthetics/public/apps/synthetics/state/monitor_management/api.ts

```diff
@@ -51,15 +51,14 @@ export const updateMonitorAPI = async ({
   return await apiService.put(`${API_URLS.SYNTHETICS_MONITORS}/${id}`, monitor);
 };
 
-export const getDecryptedMonitorAPI = async ({
-  id,
-}: {
-  id: string;
-}): Promise<DecryptedSyntheticsMonitorSavedObject> => {
-  return await apiService.get(API_URLS.GET_SYNTHETICS_MONITOR.replace('{monitorId}', id), {
-    decrypted: true,
-  });
-};
+export const getDecryptedMonitorAPI = async ({ id }: { id: string }): Promise<SyntheticsMonitor> =>
```

Comment:
Was this API updated? I thought this PR only dealt with the monitor list, but the getDecryptedMonitorAPI is for a single monitor.